.gitignore

```diff
@@ -5,3 +5,4 @@ yarn-debug.log*
 yarn-error.log*
 .nyc_output/
 .cache/
+.DS_Store
```

Comment:
See [global `.gitignore`](https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) for platform/editor specific files.